_posts/2020-3-29-managed-services-serverless.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Adding Managed Services to Serverless with Crossplane and OpenFaaS"
+description: In this post Dan from Crossplane explains how to easily deploy and consume managed services alongside your serverless functions on Kubernetes using Crossplane and OpenFaaS.
+date: 2020-03-30
+image: /images/2020-03-30-managed-services-serverless/cover.png
+categories:
+  - guest
+  - serverless
+  - kubernetes
+  - crossplane
+  - data
+author_staff_member: alex
+dark_background: true
+---
+
+I’d like to introduce you to Dan Mangum who is our guest-writer for today’s end-user blog post. Dan will be recapping a [recent live stream](https://youtu.be/XphQgB87U-s) where we showed off two demos using Crossplane with OpenFaaS to deploy and consume managed services alongside serverless functions.
+
+> Author bio: [Dan Mangum](https://twitter.com/hasheddan) is a software engineer at [Upbound](https://upbound.io/) where he works on the open source [Crossplane](https://crossplane.io/) project. He also serves on the Kubernetes release team, and is an active contributor to the Kubernetes project and multiple other open source efforts. He hosts a biweekly live stream show, [The Binding Status](https://www.youtube.com/playlist?list=PL510POnNVaaYFuK-B_SIUrpIonCtLVOzT), focused on extending Kubernetes, building Crossplane, and shaping the future of cloud-native computing.
+
+<img src="/images/2020-03-30-managed-services-serverless/mangum.jpg" width="25%" height="25%" />
+
+Kubernetes is often seen as complex and confusing for new and seasoned users alike. While it provides a wealth of tools and functionality for infrastructure operators, developers frequently struggle to navigate the complexity to just get their application running. OpenFaaS eases that burden by providing the fastest way to get your code running on Kubernetes.
+
+However, while OpenFaaS addresses the problem of getting your code to production, providing data for your functions is a different beast entirely. Most applications do not provide much value without data, so this is vital piece of the puzzle.
```

Comment:
`Most applications do not provide much value without data, so this is vital piece of the puzzle.` < disagree.

Please reframe in a more positive connotation.

Stateless can be very effective in a number of important use-cases.

Reply:
I think just removing that sentence would be appropriate here